java/src/test/java/ai/rapids/cudf/ColumnVectorTest.java

```diff
@@ -228,6 +228,78 @@ void isNullTestEmptyColumn() {
     }
   }
 
+  @Test
+  void isNanTest() {
+    try (ColumnVector v = ColumnVector.fromBoxedDoubles(1.0, 2.0, Double.NaN, 4.0, Double.NaN, 6.0);
```

Comment:
Can we have some tests that include floats not just doubles and some nulls as well?

Reply:
You got it! Coming up in the next round.